### ISSUE TITLE
Finish automated mode

### ISSUE THREAD

**Issue by romaninsh** [atk4/login#5]
Currently you need to create login page, password reminder page etc.

This needs to be consolidated into virtual pages and displayed by the `check` method similar to ATK4.3.

``` php
$auth = $app->add(['Auth', 'reminder'=>true, 'register'=>true]);
$auth->setModel(new User($app->db));
$auth->check();
```

**Comment by kvnndr0** [comment]
Rss

**Comment by romaninsh** [comment]
Implemented in 1.0
